app/domain/errors.rb

```diff
@@ -73,24 +73,24 @@ module AuthenticatorClass
 
       module Security
 
-        NotWhitelisted = ::Util::TrackableErrorClass.new(
+        AuthenticatorNotWhitelisted = ::Util::TrackableErrorClass.new(
           msg: "'{0-authenticator-name}' is not whitelisted in CONJUR_AUTHENTICATORS",
```

Comment:
Because the whitelisting happens in DB, this message ("whitelisted in CONJUR_AUTHENTICATORS") I believe is not correct. How does enabling look in the DB?

Reply:
changed to `'{0-authenticator-name}' is not enabled`